apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -113,13 +115,18 @@ export default class ScriptOverviewTopRow extends React.Component {
               assignmentName={scriptTitle}
             />
           )}
-        {experiments.isEnabled(experiments.ASSIGNMENT_UPDATES) && (
-          <SectionAssigner
-            sections={sections}
-            initialSelectedSectionId={selectedSectionId}
-            showAssignButton={showAssignButton}
-          />
-        )}
+        {!professionalLearningCourse &&
```

Comment:
This is the first time I think I've seen something restricted to non-PLC courses in the work you have been doing. Should we be restricting this on the course overview page in addition to the script page?

Reply:
My understanding is that PL courses are never actually courses only scripts so there's no work needing on the course overview page because PL stuff won't be routed there. @clareconstantine am I correct?

Reply:
I actually haven't worked with PLC courses at all - I believe @islemaster is our current expert on this.

Reply:
This is the current list of course files, in case that helps:
```
allthethingscourse.course
andrea-test-dlp.course
csd-2017.course
csd-2018.course
csd-2019.course
csd-pilot.course
csp-2017.course
csp-2018.course
csp-2019.course
fit2019-apprentice.course
```

Reply:
There is a `Plc::Course` model that requires a corresponding `Course` model, and they typically don't show up anywhere but the professional learning pages.  You can, for example, load https://studio.code.org/courses/fit2019-apprentice but I think that page is broken/unused and it might be a non-issue - I'm not sure the section assigner shows up there at all.  If it does, we probably don't want PLC courses to be assignable this way; a teacher is typically assigned to a PLC course when they are enrolled in a workshop for the course.